src/users/cd-login.vue

```diff
@@ -1,10 +1,29 @@
 <template>
-  <div class="cd-login">
-    <form @submit.prevent="login">
-      <input type="email" v-model="email" />
-      <input type="password" v-model="password" />
-      <input type="submit" value="Login" />
-    </form>
+  <div class="cd-login container-fluid">
+    <div class="row">
+      <h3 class="cd-login__header text-center">Login</h3>
+    </div>
+    <div class="row">
+      <div class="cd-login__box">
+        <form @submit.prevent="login">
+          <div class="form-group">
+            <label>Email</label>
```

Comment:
As you already mentioned, but just adding here as a reminder to both of us, all strings should be wrapped in `{{ $t(...) }}`. They also need to be added to https://github.com/CoderDojo/cp-translations if they haven't already, which I realise now I probably haven't told you... 😅